code/modules/clothing/glasses/glasses.dm

```diff
@@ -155,4 +155,29 @@
 	name = "Optical Thermal Eyepatch"
 	desc = "An eyepatch with built-in thermal optics."
 	icon_state = "eyepatch"
-	item_state = "eyepatch"
\ No newline at end of file
+	item_state = "eyepatch"
+
+/obj/item/clothing/glasses/cold
+	name = "cold goggles"

```

Comment:
except people will read this an expect them to do something against heat and cold because it explicitly says so